### ISSUE TITLE
Respect the GNU GPLv2!!

### ISSUE THREAD

**Issue by amarpreetosahan** [MiCode/Xiaomi_Kernel_OpenSource#692]
Linux is distributed under the GNU GPLv2 license, you can not distribute software without source! Xaiomi have violated the GNU GPLv2 with the distribution of Miui 8 (5.1.1) and miui 8.5 (6.0)!
I will contact the copyright owner (Linus Torvalds) to report the repeated violation of the license by Xaiomi

Thanks for the attention

**Comment by luk1337** [comment]
>I will contact the copyright owner (Linus Torvalds)

lmao

**Comment by TheStrix** [comment]
Lmao

**Comment by marines** [comment]
Nothing to laugh about. It's not as stupid as Xiaomi not complying with GPL.

**Comment by amarpreetosahan** [comment]
@TheStrix as a developer i respect you a lot, but is this really thing of laughs?

**Comment by luk1337** [comment]
@amarpreetosahan yes indeed.

**Comment by amarpreetosahan** [comment]
@luk1337 don't you think that xaiomi have to follows the license from they arw selling his product?

**Comment by luk1337** [comment]
@amarpreetosahan we weren't laughing at that, we were laughing at you.

**Comment by luk1337** [comment]
Also it looks like you are unable to spell Xiaomi right. Just FYI it's X I A O M I.

**Comment by amarpreetosahan** [comment]
Yeah yeah yeah @luk1337 you people can laughs on me as i didn't write xiaomi 's spell correctly or i said that i will tag the post to owner, bur can't you see the issue.

**Comment by akhilnarang** [comment]
🤣

**Comment by nealgosalia** [comment]
In maymay land, we say it as,
"one does not simply accuse Xiaomi of not releasing sources"

**Comment by marines** [comment]
@luk1337 you're dumb.

**Comment by insidewhy** [comment]
Xiaomi are a bunch of tossers, they don't care about the GPL. Don't buy their products. Shame as my next phone would definitely be Xiaomi if they weren't such greasy bastards.

**Comment by zieru** [comment]
1 month already, did you already contacted linus? :D

**Comment by out386** [comment]
> I will contact the copyright owner (Linus Torvalds)

Just one single owner? :rofl:

**Comment by 935713827** [comment]
在China  何为GNU GPLv2

**Comment by kiodo1981** [comment]
I hope and soon release source for Redmi 4A (rolex)
https://www.change.org/p/xiaomi-inc-publish-the-kernel-source-for-the-xiaomi-redmi-4a-4-4pro

**Comment by ghost** [comment]
There's no really restricts for Open-Source terms in China now :(

**Comment by insidewhy** [comment]
Do you open a change petition when a drug dealer rips you off in the street when you're buying weed? And yet you open one for the criminals at Xiaomi? LOL.
